frontend/app/routes/$lang+/_public+/apply+/$id+/federal-provincial-territorial-benefits.tsx

```diff
@@ -206,7 +225,8 @@ export default function AccessToDentalInsuranceQuestion() {
                           .map((option) => ({
                             children: <span className="font-bold">{getNameByLanguage(i18n.language, option)}</span>,
                             value: getNameByLanguage(i18n.language, option),
-                            defaultChecked: state?.provincialTerritorialSocialProgram === getNameByLanguage(i18n.language, option),
+                            checked: provincialProgramOption === getNameByLanguage(i18n.language, option),
+                            onChange: (e) => setProvincialProgramOption(e.target.value),
```

Comment:
In my opinion, the "Please select which social program you are covered under." legend shouldn't appear until a province is selected.

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/1daffd11-5105-41fa-a961-6612605e8914)

(Wasn't sure where to put this comment)